tlsutil/config_test.go

```diff
@@ -202,11 +203,13 @@ func startTLSServer(config *Config) (net.Conn, chan error) {
 	serverConn, _ := serverSession.Accept()
 
 	go func() {
+		serverConn.SetReadDeadline(time.Now().Add(time.Second))
```

Comment:
What's weird is that `TestConfig_outgoingWrapper_BadDC` and `TestConfig_outgoingWrapper_BadCert` would hang at the end when doing `<-errc`. It looks like in Go 1.7 the server just hangs out in `Handshake()` now, where before it would return `remote error: bad certificate`. We need to chase this down to understand why this changed. The deadline just gets it to exit with a different error.